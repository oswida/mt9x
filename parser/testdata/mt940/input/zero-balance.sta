:20:1111220206200003
:25:508765431
:28C:227/1
:60F:C111119SEK17339213,33
:61:1111211121C2496358,05NCMZCMZ501234567//6921 KOBA
:86:Zero Balancing 501234567
:61:1111211121C655344,13NCMZCMZ502345678//6921 KOBA
:86:Zero Balancing 502345678
:62F:C111121SEK20490915,51
:64:C111121SEK20490915,51