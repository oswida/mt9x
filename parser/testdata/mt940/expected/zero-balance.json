{
 "tag20": "1111220206200003",
 "tag25": {
  "account": "508765431"
 },
 "tag28": {
  "stmt_number": "227",
  "seq_number": "1"
 },
 "tag60": {
  "dc_mark": "C",
  "date": "2011-11-19T00:00:00Z",
  "currency": "SEK",
  "amount": "17339213.33"
 },
 "statements": [
  {
   "tag61": {
    "value_date": "2011-11-21T00:00:00Z",
    "entry_date": "0000-11-21T00:00:00Z",
    "dc_mark": "C",
    "amount": "2496358.05",
    "trx_ident": "NCMZ",
    "owner_ref": "CMZ501234567",
    "institution_ref": "6921 KOBA"
   },
   "tag86": [
    "Zero Balancing 501234567"
   ]
  },
  {
   "tag61": {
    "value_date": "2011-11-21T00:00:00Z",
    "entry_date": "0000-11-21T00:00:00Z",
    "dc_mark": "C",
    "amount": "655344.13",
    "trx_ident": "NCMZ",
    "owner_ref": "CMZ502345678",
    "institution_ref": "6921 KOBA"
   },
   "tag86": [
    "Zero Balancing 502345678"
   ]
  }
 ],
 "tag62": {
  "dc_mark": "C",
  "date": "2011-11-21T00:00:00Z",
  "currency": "SEK",
  "amount": "20490915.51"
 },
 "tag64": {
  "dc_mark": "C",
  "date": "2011-11-21T00:00:00Z",
  "currency": "SEK",
  "amount": "20490915.51"
 }
}